src/core/varinfo.jl

```diff
@@ -2,21 +2,28 @@ import Base.string, Base.isequal, Base.==, Base.convert
 import Base.getindex, Base.setindex!
 import Base.rand, Base.show
 
-########## VarName ##########
-
+###########
+# VarName #
+###########
 immutable VarName
   csym      ::    Symbol        # symbol generated in compilation time
   sym       ::    Symbol        # variable symbol
   indexing  ::    String        # indexing
   counter   ::    Int           # counter of same {csym, uid}
 end
 
+typealias UID Tuple{Symbol,Symbol,String,Int}
```

Comment:
I don't quite understand the problem you are meeting. But will be in the lab this afternoon, let's discuss then.